### PR TITLE
refactor UI to support supabase DB changes

### PR DESCRIPTION
**Pull request by luisflores93**
Refactor UI to support supabase db changes

### CODE REVIEW

File: src/helpers/platforms.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { GoogleMeet } from '@/components/Icons/GoogleMeet';
+
+export const platforms = {
+  'google-meet': {
+    displayName: 'Google Meet',
+    icon: <GoogleMeet />,
+  },
+  teams: {
+    displayName: 'Teams',
+    icon: <GoogleMeet />,
```
Comment (luisflores93) [inline src/helpers/platforms.tsx:10]: need other supported platforms icons as svg format

File: src/components/AvailableMeetings/MeetItem.tsx [hunk 2]
```diff
@@ -1,33 +1,28 @@
...
+import Link from 'next/link';
 import { ArrowRight } from '../Icons/ArrowRight';
 import { Clock } from '../Icons/Clock';
 import { Tag } from '../Icons/Tag';
 import { Divider } from '../common/Divider';
 
 interface Props {
-  meeting: MeetingItem;
+  meeting: EventItem;
   eventOwner: EventsOwner;
```
Comment (fiizzy) [inline src/components/AvailableMeetings/MeetItem.tsx:11]: `EventOwner`

File: src/components/Calendar/index.tsx [hunk 3]
```diff
@@ -69,14 +71,21 @@ export const Calendar = ({
...
+        selectedDate && format(selectedDate.day, 'dd');
       const isSelected = formattedSelectedDateDay === currentDay;
 
-      const hasSchedule = meetingSchedules.find((schedule) => {
+      const hasSchedule = eventsSchedules.find((schedule) => {
         return (
-          format(schedule.day, 'dd') === format(date, 'dd') &&
-          date.getMonth() === schedule.day.getMonth() &&
-          date.getFullYear() === schedule.day.getFullYear()
+          format(new Date(schedule.day).toLocaleString(), 'dd') ===
```
Comment (fiizzy) [inline src/components/Calendar/index.tsx:79]: what does `toLocaleString()` do?

File: src/helpers/platforms.tsx (hunk 1 above)
Comment (fiizzy) [reply src/helpers/platforms.tsx:10]: Want to double check if this is solved?

File: src/types/meetings.ts [hunk 4]
```diff
@@ -1,43 +1,28 @@
...
-  // will revisit this interface
-  hour: Date;
-}
-
-export interface Timezone {
-  // will revisit this interface
-  name: string;
+  eventId: number;
+  date: string;
+  isExpired?: boolean;
```
Comment (fiizzy) [inline src/types/meetings.ts:7]: Do we want to keep this as nullable?

**Review by fiizzy — COMMENTED** [review]
All looking very good @luisflores93 .
Just a few questions, and I can go ahead and merge this PR.

File: src/components/AvailableMeetings/MeetItem.tsx (hunk 2 above)
Comment (fiizzy) [reply src/components/AvailableMeetings/MeetItem.tsx:11]: Ignore this ^

File: src/components/Calendar/index.tsx (hunk 3 above)
Comment (luisflores93) [reply src/components/Calendar/index.tsx:79]: returns the date as string in the current/local timezone

Comment (fiizzy) [reply src/components/Calendar/index.tsx:79]: Perfect 👍

File: src/types/meetings.ts (hunk 4 above)
Comment (luisflores93) [reply src/types/meetings.ts:7]: I think this can be non optional(we can merge it and I'll update it in the next PR)

Comment (fiizzy) [reply src/types/meetings.ts:7]: Okay

File: src/components/Calendar/index.tsx (hunk 3 above)
Comment (luisflores93) [reply src/components/Calendar/index.tsx:79]: reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

File: src/helpers/platforms.tsx (hunk 1 above)
Comment (luisflores93) [reply src/helpers/platforms.tsx:10]: Yeah, we can add the rest on the next PRs, need the svg files from figma possibly.

**Pull request merged**